dvm-helper/internal/config/config.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+type DvmOptions struct {
+	DvmDir             string
+	MirrorURL          string
+	Token              string
+	Shell              string
+	Debug              bool
+	Silent             bool
+	IncludePrereleases bool
+	Logger             *log.Logger
+}
+
+func NewDvmOptions() DvmOptions {
+	return DvmOptions{
+		Logger: log.New(ioutil.Discard, "", log.LstdFlags),
```

Comment:
Should we set other defaults for the `MirrorURL` and others here too? Or just rely on `""` being the default string and `false` being the default for `bool`?

Reply:
I don't want to go changing too much at once, especially stuff that is unrelated to the original bug (doing a file move across disk partitions). 

I'm not terribly confident that I wouldn't cause regressions... 😊

Reply:
Makes sense.